### PR TITLE
Add DOCTYPE and lang attributes to HTML templates.

### PR DESCRIPTION
**Pull request by jbampton**
- indent HTML code
- add DOCTYPE
- add lang="en" to the HTML start tag

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: airflow/www/templates/airflow/dags.html [hunk 1]
```diff
@@ -21,11 +21,11 @@
 {% from 'appbuilder/loading_dots.html' import loading_dots %}
 
 {% block page_title %}
-  {% if search_query %}
+  {%- if search_query -%}
```
Comment (ashb) [inline airflow/www/templates/airflow/dags.html]: If you are going to do this, do it everywhere, not just in this one page. If you do it to everywhere it should be a separate PR to this one.

Honestly though: this change isn't needed - HTML is not white space sensitive, so this has little benefit.

**Review by ashb — DISMISSED** [review]
See comment

**Comment by jbampton** [comment]
Hey @potiuk or @kaxil can you please review 👍

**Review by ryanahamilton — APPROVED** [review]

**Review by potiuk — APPROVED** [review]
